x-pack/platform/test/serverless/README.md

```diff
@@ -1,118 +1,95 @@
-don't have a dedicated config file and instead need to be included in project specific
-configurations.
+Tests that are specific to individual serverless project types should be located in their respective solution directories:
 
-**If you add a new `api_integration` or `functional` `common` sub-directory, remember to add it to the corresponding `common_configs` of all projects (`x-pack/test_serverless/[api_integration|functional]/test_suites/[chat|observability|search|security]/common_configs`).**
+- **Security Solution**: `x-pack/solutions/security/test/serverless/`
+- **Observability Solution**: `x-pack/solutions/observability/test/serverless/`
+- **Search Solution**: `x-pack/solutions/search/test/serverless/`
+- **Other Solutions**: `x-pack/solutions/<solution>/test/serverless/`
 
-In case a common test needs to be skipped for one of the projects
-(in both regular pipelines that start KBN in serverless mode [against serverless ES] & pipelines creating serverless projects in MKI [Cloud]),
-there are the following suite tags available to do so:
-`skipSvlChat`, `skipSvlOblt`, `skipSvlSearch`, `skipSvlSec`, which can be added like this to a test suite:
+**Guidelines for platform tests**:
 
-```ts
-describe('my test suite', function () {
-  this.tags(['skipSvlChat', 'skipSvlOblt', 'skipSvlSearch', 'skipSvlSec']);
-  // or for a single tag: this.tags('skipSvlSec');
-  [...]
-});
-```
+- Focus on functionality that works identically across all project types
+- Test core platform features (spaces, saved objects, user management, alerting, etc.)
+- Validate shared UI components and navigation patterns
+- Ensure consistent behavior across different serverless project types
 
-Tests that are designed to only run in one of the projects should be added to the project
-specific test directory and not to `common` with three skips.
+**What should NOT be included**:
 
-Note, that `common` tests are invoked up to four times in a full test run: once per project to make
-sure the covered shared functionality works correctly in every project. So when writing tests there, be mindful about the test run time.
+- Solution-specific features or APIs
+- Project-type specific UI elements or workflows
+- Business logic specific to Security, Observability, Search, or Chat solutions
 
-See also the README files for [Serverless Common API Integration Tests](https://github.com/elastic/kibana/blob/main/x-pack/platform/test/serverless/api_integration/test_suites/README.md) and [Serverless Common Functional Tests](https://github.com/elastic/kibana/blob/main/x-pack/platform/test/serverless/functional/test_suites/README.md).
+### Shared Services and Page Objects
 
-### Shared services and page objects
-
-Test services and page objects from `x-pack/test/[api_integration|functional]`
+Test services and page objects from `x-pack/platform/test/[api_integration|functional]`
 are available for reuse.
```

Comment:
Just in the context of platform-shared tests or for solution-specific too?

Reply:
By shared we mean "platform" here, this structure is not guaranteed in solutions